Gulpfile.js

```diff
@@ -35,38 +35,38 @@ gulp.task('sass', function() {
   console.log('Running Sass and PostCSS...');
   // Set what postcss plugins need to be run
   var processors = [
-    autoprefixer(defaults.autoprefixer),
+    autoprefixer('last 2 versions', '> 5%'),
     cssnano
   ];
-  // Run on all file defaults defined in var scsss
-  return gulp.src(defaults.scss)
+  // Run on all file defaults defined in var scss
+  return gulp.src(['../../styleguide/source/code/sass/*.scss', '../../styleguide/source/code/sass/**/*.scss'])
```

Comment:
I'm not sure how or why this got changed from being the variable, but that needs to go back to using the defaults variable so that it can be overridden.